packages/core-blocks/src/index.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	registerBlockType,
+	setDefaultBlockName,
+} from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import * as paragraph from './paragraph';
+import * as image from './image';
+import * as heading from './heading';
+import * as quote from './quote';
+import * as gallery from './gallery';
+import * as archives from './archives';
+import * as audio from './audio';
+import * as button from './button';
+import * as categories from './categories';
+import * as code from './code';
+import * as columns from './columns';
+import * as column from './columns/column';
+import * as coverImage from './cover-image';
+import * as embed from './embed';
+import * as file from './file';
+import * as latestComments from './latest-comments';
+import * as latestPosts from './latest-posts';
+import * as list from './list';
+import * as more from './more';
+import * as nextpage from './nextpage';
+import * as preformatted from './preformatted';
+import * as pullquote from './pullquote';
+import * as reusableBlock from './block';
+import * as separator from './separator';
+import * as shortcode from './shortcode';
+import * as spacer from './spacer';
+import * as subhead from './subhead';
+import * as table from './table';
+import * as textColumns from './text-columns';
+import * as verse from './verse';
+import * as video from './video';
+
+export const registerCoreBlocks = () => {
```

Comment:
So we have it duplicated in here but without two problematic blocks. Interesting.